cluster/stmt.go

```diff
@@ -71,6 +71,7 @@ func StmtString(code int) (string, error) {
 	return "", fmt.Errorf("No prepared statement registered with code %d", code)
 }
 
+// Deprecated.
```

Comment:
I think the format of this needs to be something like this to mark things as deprecated in Go docs:

```golang
// GetCallerProject will get the go project name of whichever function called `GetCallerProject`.
//
// Deprecated: The caller project is no longer required and causes issues when vendoring.
// This is no longer required when calling `PrepareStmts`.
```

https://go.dev/blog/godoc